### DISCUSSION TITLE
How can I create users

### DISCUSSION THREAD

**Discussion by happyDuke96** [chatwoot/chatwoot#5671] [Chatwoot APIs]
Hello, I built my server and I think I set it up correctly, I logged into my account with api and received a token, I manage to create agents but I can’t create an account and I can’t give a password to users, and when using platform urls I get this error
{
"error": "Invalid access_token"
}
I need to create users (agents and administrators) I have to give them a role and give them a password after verification can you help? any information would be helpful

**Comment by sojan-official** [comment]
For using the platform API token, you will have to create a platform app in super admin. These APIs are only available in self-hosted.
https://www.chatwoot.com/docs/self-hosted/monitoring/super-admin-sidekiq/

**Reply by happyDuke96** [reply]
![photo_2022-10-20_09-50-31](https://user-images.githubusercontent.com/105979703/196859098-4fce6ebb-2be3-4cc5-9da1-86be6842b928.jpg)
![photo_2022-10-20_09-50-29](https://user-images.githubusercontent.com/105979703/196859102-c6a9e86c-de3f-4631-8f63-dc6038415497.jpg)

I received this token when signing up

{
    "error": "Invalid access_token"
}

as a result I get this error,I installed via VM Linux not some nginx I didn't use

**Reply by happyDuke96** [reply]
I understood everything, I successfully received a token from the platform, now I have this error
through the browser it is accurate with such an email and the password is registered but does not give through the api
![Снимок](https://user-images.githubusercontent.com/105979703/196887419-270de47e-d7f9-4730-ae74-9c055e16d1f9.PNG)

**Comment by happyDuke96** [comment]
Using api, I can't delete the users that I created via the web browser,there is a little discrepancy in the cards, but I entered it correctly
![Снимок2](https://user-images.githubusercontent.com/105979703/196888633-81ae8b9c-4be8-400e-9a65-3a8aa72f9bbc.PNG)

**Comment by happyDuke96** [comment]
problem solved thank you very much
